packages/controls/src/controls/group/__snapshots__/group.test.ts.snap

```diff
@@ -1,18 +1,21 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`ShapeV2 Shape data is gracefully handles missing/extra props copyData 1`] = `
+exports[`Group Group data is gracefully handles missing/extra props copyData 1`] = `
```

Comment:
Updated `ShapeV2` test snapshots. Mostly snapshot name changes, highlighting other noteworthy diffs below.

Reply:
Best reviewed with the "Hide whitespace" setting on.